### PR TITLE
FAT-2649/C350421

### PR DESCRIPTION
**Pull request by re-jas**
[Implement e-2-e automation of test case C350421: Check that user can create Item level request from Item record](https://issues.folio.org/browse/FAT-2649)
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1464/allure/

### CODE REVIEW

File: cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      addedCirculationRule = 't ' + testData.loanTypeId + ': i ' + ruleProps.i + ' l ' + ruleProps.l + ' r ' + ruleProps.r + ' o ' + ruleProps.o + ' n ' + ruleProps.n;
+      CirculationRules.addRuleViaApi(originalCirculationRules, ruleProps, 't ', testData.loanTypeId);
+    });
+
+    cy.createTempUser(
+      [
+        permissions.uiUsersfeefinesCRUD.gui,
+        permissions.uiUsersfeefinesView.gui,
+        permissions.inventoryAll.gui,
+        permissions.loansRenew.gui,
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js:110]: do you need this permission?

File: cypress/support/fragments/requests/newRequest.js [hunk 2]
```diff
@@ -93,5 +93,49 @@ export default {
...
+    cy.expect([
+      Accordion('Item information').exists(),
+      Accordion('Request information').exists(),
+      Accordion('Requester information').exists()
+    ]);
+  },
+
+  enterItemInfo(barcode) {
+    cy.do(itemBarcodeInput.fillIn(barcode));
+    cy.do(enterItemBarcodeButton.click());
```
Comment (penfriend) [inline cypress/support/fragments/requests/newRequest.js:110]: please write cy.do([]) or cy.do...then(()=> cy.do

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
+import uuid from 'uuid';
+import TestTypes from '../../support/dictionary/testTypes';
```
[the remark this answers is not in the document]
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js:2]: done

[the remark this answers is not in the document]
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js:2]: done

File: cypress/support/fragments/requests/newRequest.js [hunk 4]
```diff
@@ -93,5 +93,49 @@ export default {
...
+  },
+
+  checkErrorMessage(message) {
```
[the remark this answers is not in the document]
Comment (re-jas) [reply cypress/support/fragments/requests/newRequest.js:113]: done

File: cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-create-item-level-request-from-item-record.cy.js:110]: removed

File: cypress/support/fragments/requests/newRequest.js (hunk 2 above)
Comment (re-jas) [reply cypress/support/fragments/requests/newRequest.js:110]: done

**Review by penfriend — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**